heartbeat/interface.go

```diff
@@ -16,8 +16,10 @@ import (
 // P2PMessenger defines a subset of the p2p.Messenger interface
 type P2PMessenger interface {
 	Broadcast(topic string, buff []byte)
+	BroadcastWithSk(topic string, buff []byte, pid core.PeerID, skBytes []byte)
```

Comment:
Can we raname this into `BroadcastUsingSk` (or `BroadcastSigningWithSk`)? I know that the naming convention follows libp2p's `PublishWithSk` but I find it very misleading, I really thought we were sending the sk over the wire. (`SendableData` having an SK member doesn't help either :)) @iulianpascalau thoughts?)

Reply:
right, let's rename to `BroadcastWithPrivateKey`

Reply:
pushed